app/app.rb

```diff
@@ -47,7 +46,7 @@ def create_student
 
     print 'Age: '
     age = gets.chomp
-    binding.pry
+
```

Comment:
- [ ] because there are no changes made I can't leave accurate inline comments, however, all requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input.
- [ ] in lines 57/58 and 72/73 you're repeating the same line of code this is not in line with the DRY principle.